src/components/ExperimentsList.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react'
+// import { Link } from 'react-router-dom'
+import { Container, Row, Col, Button, Card, ListGroup, ListGroupItem } from "reactstrap";
+import "../static/css/Experiments.css";
+ 
+
+const ExperimentsList = () => {
+
+    const [data] = useState([
+        { id: "1", emojis: "🚀💸", name:"Up Only WBTC" },
+        { id: "2", emojis: "⏰🏦", name:"WBTC Theta Vault" },
+        { id: "3", emojis: "🎢🤑", name:"Vega Vault" }
+      ]);
```

Comment:
I see we're duplicating the [data here](https://github.com/ribbon-finance/experiments-site/pull/7/files#diff-f366a894c764c23476d1897fda76dff47fe3e9b8206bba3209832011ebb782b4R7), is this a redundant component?